docs/core/Graph.md

```diff
@@ -39,6 +39,7 @@ Graph is a set of graphs that share a common set of base visualization structure
 | dimension              | object   | {}                                             | Refer [Dimension](#dimension)                                                                                                                                                               |
 | legendPadding          | object   | `{ top: 4, bottom: 4, left: 8, right: 8 }`     | Sets the legend padding and removes the carbon default margin of 8px around legend                                                                                                          |
 | locale                 | object   | LOCALE.en_US                                   | Locale object for X-Axis tick values                                                                                                                                                        |
+| opaqueBackground       | false    | false                                          | Toggle between a transparent or White background for the container                                                                                                                          |
```

Comment:
Can you add dev example for these changes also?

Reply:
Added here: https://github.com/cerner/carbon-graphs/pull/274/commits/e37645be9b2a529b5af3ae0a723a309aaafc63d0